src/buffer.rs

```diff
@@ -421,10 +423,13 @@ mod ops {
                 (CellBuffer::UInt8(lhs), CellBuffer::UInt8(rhs)) => Ord::cmp(&lhs, &rhs),
                 (CellBuffer::UInt16(lhs), CellBuffer::UInt16(rhs)) => Ord::cmp(&lhs, &rhs),
                 (CellBuffer::UInt32(lhs), CellBuffer::UInt32(rhs)) => Ord::cmp(&lhs, &rhs),
+                #[cfg(not(feature = "gdal"))]
                 (CellBuffer::UInt64(lhs), CellBuffer::UInt64(rhs)) => Ord::cmp(&lhs, &rhs),
+                #[cfg(not(feature = "gdal"))]
                 (CellBuffer::Int8(lhs), CellBuffer::Int8(rhs)) => Ord::cmp(&lhs, &rhs),
                 (CellBuffer::Int16(lhs), CellBuffer::Int16(rhs)) => Ord::cmp(&lhs, &rhs),
                 (CellBuffer::Int32(lhs), CellBuffer::Int32(rhs)) => Ord::cmp(&lhs, &rhs),
+                #[cfg(not(feature = "gdal"))]
                 (CellBuffer::Int64(lhs), CellBuffer::Int64(rhs)) => Ord::cmp(&lhs, &rhs),
```

Comment:
I'm not happy with this blanket disabling types based on gdal. I'll probably come back to it at some point and fix properly (based on actual gdal version), but it requires writing a `build.rs` script and I wanted to put that off.

Reply:
^^^ no longer relevant